pkg/gohai/gohai_test.go

```diff
@@ -53,11 +53,6 @@ type gohaiPayload struct {
 		MountedOn string `json:"mounted_on"`
 		Name      string `json:"name"`
 	} `json:"filesystem"`
-	Memory struct {
-		// SwapTotal is not reported on Windows
-		SwapTotal string `json:"swap_total"`
-		Total     string `json:"total"`
-	} `json:"memory"`
```

Comment:
The memory tests were moved directly in the memory package.

Testing the global `Collect` method is not interesting as it's never used except if Gohai is used as a binary.